dashboard/app/assets/stylesheets/application.scss

```diff
@@ -3390,9 +3390,6 @@ button.button-active-no-border:active {
   // an active button. (overriding button:active rule above)
   border: none !important;
 }
-button:hover {
-  @include box-shadow(2px 2px 5px $shadow);
-}
```

Comment:
Should we go ahead and add a follow-up item to change to a different color for the continue/finish button onHover in aichat levels? Sorry - should've asked this question earlier.